### ISSUE TITLE
Dockerfile and requirements.txt for reproducibility

### ISSUE THREAD

**Issue by leelasd** [isayevlab/AIMNet2#3]
Hi, 

I am opening this issue to add the `requirements.txt` and Dockerfile for containerizing this awesome repo :)

**Comment by zubatyuk** [comment]
Thanks for the example docker file.
Two questions:
 - requirements.txt: you did not provide this file and do not use it in the Dockerfile
 - the base docker image is mambaforge. But you use pip to install Python packages. Please confirm you believe that this is the right way. In my experience, pip-installed packages have greater chances of getting into conflict. 

Overall, I'm not sure if Dockerfile belongs to this repository. I will think about this. The main goal here is to provide models and examples, not to build a re-distributable package.

**Comment by leelasd** [comment]
Oops, sorry, I added the requirements file. I realized it was easier to directly add them to the Dockerfile since it only has a few dependencies. 

I used pip to install packages that are well defined as using mamba/conda with large channels like conda-forge takes forever to install any python package. If we want more strict versioning of dependencies, we can add the version numbers to PyPI. 
Overall, Pypi is much faster, and if you are using it on AWS/GCP cloud resources, using Conda can slow you down quite a lot. 

It's totally up to you if you want to keep the Dockerfile or not. I found the repo very useful, and Dockerizing the repo is the best way to run the code on my system without disturbing the existing packages. So, I wanted to share that. Feel free to reject/close the PR.
